packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -179,6 +179,37 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
     }
   }
 
+  const renderFeeAmount = (quote?: SimplexQuote | ProviderQuote[]) => {
+    const feeAmount = getLowestFeeValueFromQuotes(quote)
+
+    if (feeAmount === undefined) {
+      return '-'
+    }
+
+    if (feeAmount === 0) {
+      return t('global:free')
+    }
+
+    return (
+      <CurrencyDisplay
+        amount={{
+          value: 0,
+          localAmount: {
+            value: feeAmount,
+            currencyCode: localCurrency,
+            exchangeRate: 1,
+          },
+          currencyCode: localCurrency,
+        }}
+        hideSymbol={false}
+        showLocalAmount={true}
+        hideSign={true}
+        showExplicitPositiveSign={false}
```

Comment:
Small nits:
- I think the default for `hideSymbol` and `showExplicitPositiveSign` is already false, so it's not necessary to set them explicitly unless you think it helps for clarity.
- `style` is receiving an array which is unnecessary since it only has one element

I see this was already like this from before, but just figured I'd mention it